src/__tests__/components/Home.test.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import Home from '../../components/Home/Home';
+import { Home, mapDispatchToProps } from '../../components/Home/Home';
+
+const props = {
+  location: {
+    search: '?token=jfmvnvvmvmvncvmcmnvcmmvm',
+  },
+  getCurrentUser: jest.fn().mockImplementation(() => Promise.resolve({ status: 200 })),
+};
 
 describe('<Home />', () => {
   test('Should render the <Home />', () => {
-    const wrapper = shallow(<Home />);
+    const wrapper = shallow(<Home {...props} />);
     expect(wrapper).toMatchSnapshot();
   });
```

Comment:
where's the empty line?

Reply:
@kimotho-njoki Thank you! let me add the empty line